Extension/walkthrough/run-and-debug-project-windows.md

```diff
@@ -0,0 +1,13 @@
+# Run and debug your C++ file on Windows
+
+1. Open the C++ source file that you want to run and debug. Make sure this file is active (currently displayed) in the editor.
+
+2. From the main menu, select **Run > Add Configuration...** and choose **C++ (Windows)**.
```

Comment:
This only works if launch.json doesn't exist already (that might be a bug)...not sure if that matters in the context of this walkthrough though.

Reply:
This is a good thing to think about when we measure the success of the walkthrough on new users vs. returning users. New users likely won't run into the issue of already having a launch.json, but returning users might! Ideally, we'd automatically mark this step as done if we detect a launch.json in their workspace. VS Code is adding support for that, it's not currently available. Let's keep the walkthrough as is, since we're targeting new users, but keep an eye on what returning users do at this step.

Reply:
@jureid New users can easily hit this due to bug https://github.com/microsoft/vscode-cpptools/issues/7517 . They may not immediately know which compiler to select when the list appears, so when they cancel the empty launch.json will get created...maybe that's a reason to fix the issue to not create any launch.json, but I don't know if that's possible yet.